test/support/generators.ex

```diff
@@ -46,6 +62,10 @@ defmodule Gnat.Generators do
 
   def payload(size), do: binary(size)
 
+  def protocol_message do
+    union([ok(), ping(), pong(), error(), message()])
```

Comment:
I think INFO might be good to add here.

Reply:
:+1: you are totally right, just forgot about it, but we definitely parse it. Will add